include/envoy/router/route_config_update_receiver.h

```diff
@@ -73,6 +78,11 @@ class RouteConfigUpdateReceiver {
    * @return SystemTime the time of the last update.
    */
   virtual SystemTime lastUpdated() const PURE;
+
+  /**
+   * @return aliases received with the last VHDS update.
```

Comment:
Ditto; I'm wondering how this works across multiple updates? 

Also, who are the aliases for? A VHDS update has multiple virtual hosts, each of which can have aliases?

Reply:
> A VHDS update has multiple virtual hosts, each of which can have aliases?

Yep.

Reply:
> Is it the union of all the names/alises for all virtual hosts

yep